framework/src/auxkernels/MaterialRateRealAux.C

```diff
@@ -0,0 +1,42 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "MaterialRateRealAux.h"
+
+registerMooseObject("MooseApp", MaterialRateRealAux);
+registerMooseObject("MooseApp", ADMaterialRateRealAux);
+
+template <bool is_ad>
+InputParameters
+MaterialRateRealAuxTempl<is_ad>::validParams()
+{
+  InputParameters params = MaterialAuxBaseTempl<Real, is_ad>::validParams();
+  params.addClassDescription("Outputs element volume-averaged material properties rate of change");
```

Comment:
where is that "volume averaged" stuff coming from. This is misleading. I'd just remove it.